src/shared/credentialsStore.ts

```diff
@@ -0,0 +1,53 @@
+import * as keytarType from 'keytar';
+import { AuthParams } from './socket';
+
+function getNodeModule<T>(moduleName: string): T | undefined {
+    const vscodeRequire = eval('require');
```

Comment:
Why do we need to use `eval()` here?

Reply:
I adapted this code from another extension using the keychain; here:  https://github.com/microsoft/vscode-pull-request-github/blob/master/src/authentication/keychain.ts#L8-L15

To my knowledge, we do not want to directly require keytar due to it being a native module. Instead, we want to force it to use the keytar that is already shipped with VS Code/built for VS Code. I believe this `eval('require')` is what accomplishes this. It means it will not evaluate the require until run-time and at that time and, because of that, it will use VS Code's keytar.

Providing our own keytar requires building the native module ourselves, which I found to mismatch the electron version, environment of VS Code.